web/themes/custom/contribtracker/cypress.config.js

```diff
@@ -1,8 +1,56 @@
 import { defineConfig } from 'cypress';
+import { readFileSync } from 'fs';
+import neatCSV from 'neat-csv';
+import path from 'path';
+import cypressGrep from 'cypress-grep/src/plugin';
+// import cypressSplit from "cypress-split"; // Uncomment if you are using cypress-split
 
 export default defineConfig({
+  projectId: 'contrib-tracker',
+  viewportWidth: 1512,
+  viewportHeight: 1200,
+  watchForFilesChanges: false,
+  chromeWebSecurity: false,
+  defaultCommandTimeout: 60000,
+  execTimeout: 10000,
+  pageLoadTimeout: 500000,
+  requestTimeout: 90000,
+  responseTimeout: 90000,
+  experimentalStudio: true,
+  trashAssetsBeforeRuns: false,
+  experimentalShadowDomSupport: true,
+  reporter: 'mochawesome',
```

Comment:
How does this reporter help? I'd like to keep dependencies to a minimum if this is not important.